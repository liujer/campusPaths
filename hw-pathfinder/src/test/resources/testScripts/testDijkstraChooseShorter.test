# Tests if Dijkstra's algorithm will choose the shorter path
# in terms of number of nodes in the path
CreateGraph A
AddNode A one
AddNode A two
AddNode A three

AddEdge A one two 1
AddEdge A two three 1
AddEdge A one three 1

FindPath A one three