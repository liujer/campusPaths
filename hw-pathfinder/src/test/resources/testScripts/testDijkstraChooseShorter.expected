# Tests if Dijkstra's algorithm will choose the shorter path
# in terms of number of nodes in the path
created graph A
added node one to A
added node two to A
added node three to A

added edge 1.000 from one to two in A
added edge 1.000 from two to three in A
added edge 1.000 from one to three in A

path from one to three:
one to three with weight 1.000
total cost: 1.000